### DISCUSSION TITLE
Why not Hero 5 black support? & can you make the camera beep?

### DISCUSSION THREAD

**Discussion by NeedNot** [gopro/labs#326] [Q&A] [answered]
I have a hero 5 black and a max and i think labs sounds cool but could you for example make it detect if the video is all black or very dark to start beeping?

i mostly record in a lake that is 0% see through but is only 5.5ft deep so if somehow i dropped the camera and it was lost it would start beeping and i could easily find it. this is needed because wifi and BT dont work underwater

**Comment by dnewman-gpsw** [comment]
It is possible in the latest HERO11 firmware to create a script that does that, beep when light level is very low (although I haven't tried that particular hack combination.) However this in not coming to HERO5.  The oldest Black edition camera supported is HERO7.  Labs team is very busy maintaining and updating for current cameras, that leaves no bandwidth for older cameras.

**Reply by NeedNot** [reply]
alright so not possible with the max at least? maybe it could detect when no gps signal is present? but i dont see anything in labs about making it beep

**Reply by dnewman-gpsw** [reply, marked as answer]
MAX got an update yesterday. It should have new beeps controls, !Bx (see https://gopro.github.io/labs/control/actions/). However, it doesn't have the light metering controls for this particular hack would not work.   You could use other sensors, like beep when motion is stopped after x seconds, etc.

**Reply by NeedNot** [reply]
oh i didn't think about that! i just threw the motion out since i dont watch motion but i could do the opposite. ill give it a try
